cpp/src/io/parquet/page_enc.cu

```diff
@@ -60,6 +60,12 @@ using ::cudf::detail::device_2dspan;
 
 constexpr uint32_t rle_buffer_size = (1 << 9);
 
+// do not truncate statistics
+constexpr int32_t NO_TRUNC_STATS = 0;
+
+// minimum scratch space required for encoding statistics
+constexpr size_t min_stats_scratch_size = sizeof(__int128_t);
+
```

Comment:
Similar to my comment above: it is recommended to use ALL_CAP for constant names.